### ISSUE TITLE
Chapter 3, Section 1

### ISSUE THREAD

**Issue by axello** [editorscut/ec009-SwiftUIKickstart#21]
Hi Daniel,
Great book so far.

But in Chapter 3, Section 1, you start with 
```
“Create a new iOS Single View project named HelloSwiftUI with the User Interface set to SwiftUI.”
```
However, there *is* already a project with the same name.
Then, in chapter 4, you 'continue' with the 'Numbers' project.

**Comment by axello** [comment]
Whoops, after careful inspection, it seems I missed the line in 3.8 to create a new project. Mea culpa.

**Comment by editorscut** [comment]
Did you stumble on this enough that I should say more. i.e. it shouldn't IMO require careful inspection. If so, please feel free to re-open. And thank you for taking the time

**Comment by axello** [comment]
It only happened once. But my mode of working through a book like this is different: I do the exercises and experiment at the same time. So it's a bit chaotic but very educational.
So perhaps I missed the line to create a new app and I simply added the valueslider to the original HelloSwiftUI project?
Maybe start with a clean new sub chapter, to differentiate between buttons & text and the slider?

Oh, you do that. 

I think I bisected my problem: I did not read the sentence `“Create a new iOS Single View app using SwiftUI and name it Numbers.”`, around page 243.
When you then continue reading with `“Add a SwiftUI View file for the Slider named ValueSlider.swift and another for the Text named ValueView.swift.”`, everything works, sort of (I figured it out), but I never started the new project.
